packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/nodejs/variables.md

```diff
@@ -0,0 +1,4 @@
+Line 38: `var accountSid = process.env.ACCOUNT_SID;`<br>
```

Comment:
In terms of maintenance, I'd be a little worried about using line numbers here. (This might be better documented in the code as a comment, e.g., // The value for this variable is referenced from the .env file